client/src/components/playercard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import axios from 'axios';
+
+
+class PlayerCard extends React.Component {
+    constructor() {
```

Comment:
remember you can leave out the constructor and super here 

you can do this

state = {
player: []
}